godtools/App/Share/Data/FavoritedResourcesRepository/Cache/Models/RealmFavoritedResource.swift

```diff
@@ -13,14 +13,9 @@ class RealmFavoritedResource: Object {
     
     @objc dynamic var createdAt: Date = Date()
     @objc dynamic var resourceId: String = ""
+    @objc dynamic var position: Int = 0
```

Comment:
Something that just came to mind here is user's who already have the app installed all of their favorited tools will be at position 0 which could cause some issues.

One route is to make the position optional and then first sort by position and fallback to sorting by createdAt date for positions that are null.

Reply:
I ended up going the route to validate each time a fetch occurs.  A user that already has the app installed wouldn't be able to reorder if we didn't "correct" the positions at some point (unless they removed and re-added their favorites).